src/sampler/status.jl

```diff
@@ -53,24 +45,22 @@ end
 
 
 struct FullStatusSampler <: AbstractStatusSampler
-    data::Dict
+    E::Int
+    G::Int
 end
```

Comment:
What is that?

Reply:
`FullStatusSampler` is the default which keeps all components enabled